test/ec_internal_test.c

```diff
@@ -16,6 +16,7 @@
 #include "testutil.h"
 #include <openssl/ec.h>
 #include "ec_local.h"
+#include "../bn/bn_local.h"
```

Comment:
I would really like to not see this include here..
Would making an internal function for
ossl_bn_mont_ctx_eq(const BN_MONT_CTX *m1, const BN_MONT_CTX *m2) be sufficient to do this?

Reply:
Yeah, that should be enough.